### DISCUSSION TITLE
Demo of my stand-alone keyboard

### DISCUSSION THREAD

**Discussion by MichaelPNolan** [marcel-licence/esp32_basic_synth#69] [Show and tell]
This is a quick youtube - because I've been a bit blocked getting one made so I rushed this basic one together.
https://youtu.be/GGS7KVTav10

This is the state of my use of Marcel code here into my own fork. ESP32, gpio expander with a keypad library to make the toy keys trigger notes. Its not using midi but I do want to put a midi in/out on the next iteration. Also some work is mostly complete on an arpeggiator module. That could lead into a more complicated set of features to do sequencing.

At the end I talk about another toy I made (midi controller/arpeggiator) but camera work is shite. This is it https://pbs.twimg.com/media/E6Pyv2GVcAMo3-e?format=jpg&name=large

Now I bought an old casiotone at the markets and I'm going to try and wire that up to get a better keybed and more space to put stuff inside the box.

**Comment by marcel-licence** [comment]
Hey thank you for your video. I can fully agree, the parts are not expensive but you always need a lot of time.
